lenses_python/TopicHandler.py

```diff
@@ -119,28 +113,31 @@ def CreateTopic(self, topicName, replication, partitions, config, filename):
                 config = temp_dict["config"]
             else:
                 raise Exception("Failed to read configuaration file\n")
-        headers = {'Content-Type': 'application/json', 'Accept': 'text/plain',
+
+        headers = {'Content-Type': 'application/json', 'Accept': '*/*',
                    'x-kafka-lenses-token': self.token}
-        url = self.url+"/api/topics"
+
         params = dict(topicName=topicName,
-                      replication=int(replication),
-                      partitions=int(partitions),
+                      replication=int(par_rep['replication']),
+                      partitions=int(par_rep['partitions']),
                       configs=config
                       )
-        response = post(url, headers=headers, json=params)
+
+        response = post(self.url + "/api/topics", headers=headers, json=dumps(params))
+
         if response.status_code != 201:
             raise Exception("Http status code {}.{}".format(response.status_code, response.text))
 
-    def DeleteTopic(self, topicname):
+    def DeleteTopic(self, **kopts):
```

Comment:
I used **kopts because kopts functions is not ready, hence the ManageTopics functions which calls those handlers, passes always an argument, event if it is an empty list.

When done with kopts function, we can include cases based on the action and hence remove all **kopts from functions like the above.